src/Photos.vue

```diff
@@ -67,9 +68,18 @@ export default {
 			loading: true,
 			svgplaceholder,
 			imgplaceholder,
-			videoplaceholder,
+			videoplaceholder
 		}
 	},
+	computed: {
+		mapsLink() {
+			if (isMapsInstalled) {
+				return 'apps/maps'
```

Comment:
You should return thesame as the original object: `{name: 'maps'}`

Reply:
1. How can I get the individual server path? http(s)://host/path/index.php
2. How do I have to change the router to navigate to the new route? It appends the route path only on the existing route in the browser

Thanks!

Reply:
you need to use the generateUrl method for both.
Ignore my first comment about `{name: 'maps'}` I got confused about the old code :)
What you did was good, but you need to use the generateUrl.

Reply:
https://github.com/nextcloud/photos/blob/b2c8850d058551e71fa672e7c0002ef4f02ca65c/src/router/index.js#L23
https://github.com/nextcloud/photos/blob/b2c8850d058551e71fa672e7c0002ef4f02ca65c/src/router/index.js#L37